.github/workflows/java-gradle-plugin.yaml

```diff
@@ -83,26 +83,41 @@ jobs:
     name: Test
     runs-on: ubuntu-22.04
     needs: build
-
     steps:
-      - name: Test
-        uses: bakdata/ci-templates/actions/java-gradle-test@v1.16.0
+      - name: Run unit tests
+        uses: bakdata/ci-templates/actions/java-gradle-test@feat/split-test
+        # uses: bakdata/ci-templates/actions/java-gradle-test@1.35.0
         with:
-          sonar-token: ${{ secrets.sonar-token }}
-          sonar-organization: ${{ secrets.sonar-organization }}
-          signing-secret-key-ring: ${{ secrets.signing-secret-key-ring }}
-          signing-key-id: ${{ secrets.signing-key-id }}
-          signing-password: ${{ secrets.signing-password }}
+          gradle-cache: ${{ inputs.gradle-cache }}
+          gradle-version: ${{ inputs.gradle-version }}
           java-distribution: ${{ inputs.java-distribution }}
           java-version: ${{ inputs.java-version }}
-          gradle-version: ${{ inputs.gradle-version }}
+          working-directory: ${{ inputs.working-directory }}
+
+  assess-code:
```

Comment:
In order to not repeat the build, test and assess step over and over again, we should have a java-gradle base workflow which has these three steps and is called by these three workflows. We do the same at our customer, @yannick-roeder can show you if there are questions

Reply:
alright makes sense. am on it...